test/spec/desktop/viewportSize.js

```diff
@@ -21,4 +21,14 @@ describe('setViewportSize/getViewportSize', function() {
         });
     });
 
+    it('should set window size exactly when parameter \'type\' is false', function() {
+        return this.client.setViewportSize({
+            width: 500,
+            height: 500
+        }, false).windowHandleSize().then(function(res) {
```

Comment:
scratch that above .. we should test if it changes the window size if no type parameter was assigned and viewport size if type parameter is true